app/controllers/dailies_controller.rb

```diff
@@ -2,6 +2,7 @@ class DailiesController < ApplicationController
 
   def index
     @dailies = current_user.dailies
+    @chart_data = @dailies.map { |daily| daily.table_row}.reduce(:+)
```

Comment:
@case-eee I'm using a ruby enumerable to collect the data the chart needs, but I realize this will result in inefficient multiple queries. Do you think I should try to get the data in a single query?